x-pack/legacy/plugins/canvas/external_runtime/api/obj_embed.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import 'whatwg-fetch';
```

Comment:
I'll test that.  I added it here because it's a polyfill for `fetch` in browsers that don't have it as part of the browser API.  There's no export you actually use... it's an import-once-apply-polyfill gig.